packages/viewer/src/components/scene-tree/lib/controller.ts

```diff
@@ -746,6 +746,19 @@ export class SceneTreeController {
         }
       });
 
+      stream.on('status', (s) => {
+        if (s.code !== 0) {
+          console.error(
+            `Failed to load scene tree with code=${s.code}, details=${s.details}`
```

Comment:
Let's reword to `Failed to subscribe to scene tree with code=${s.code}, details=${s.details}`